docs/next/package.json

```diff
@@ -26,6 +26,7 @@
     "@tailwindcss/typography": "^0.4.0",
     "add": "^2.0.6",
     "autoprefixer": "^10.1.0",
+    "axios": "^0.26.0",
```

Comment:
Is this because we don't have access to a real `fetch` implementation?

Reply:
It's just axios avoids boilerplate `fetch` would need